### PR TITLE
feature/flaresolverr

### PR DESCRIPTION
**Pull request by wajeht**
FlareSolverr is a proxy server to bypass Cloudflare protection

### ☑️ Self Check before Merge

- [x] I have tested the template using the method described in README.md thoroughly
- [x] I have ensured that I put as much default values as possible (except passwords) to ensure minimum effort required for end users to get started.
- [x] I have ensured that I am not using the "latest" tag as this tag is dynamically changing and might break the one-click app. Use a fixed version.
- [x] I have made sure that instructions.start and instructions.end are clear and self-explanatory.
- [x] Icon is added as a png file to the logos directory.
- [x] I've executed the checks if necessary by running `npm ci && npm run validate_apps && npm run formatter` (If failling run the prettier: `npm run formatter-write`)
- [x] I will take responsibility addressing any issues that arises as a result of this PR (maintaining this app).

### CODE REVIEW

**Comment by wajeht** [comment]
@githubsaturn 

I've tested it here: https://flaresolverr.jaw.dev/

File: public/v4/apps/flaresolverr.yml [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+            CAPTCHA_SOLVER: $$cap_captcha_solver
+            TZ: $$cap_tz
+            LANG: $$cap_lang
+            HEADLESS: $$cap_headless
+            BROWSER_TIMEOUT: $$cap_browser_timeout
+            TEST_URL: $$cap_test_url
+            PROMETHEUS_ENABLED: $$cap_prometheus_enabled
+            PROMETHEUS_PORT: $$cap_prometheus_port
+        ports:
+            - $$cap_port:8191
```
Comment (githubsaturn) [inline public/v4/apps/flaresolverr.yml:18]: Why do we need to map the port? `containerHttpPort` should be enough, no?

File: public/v4/apps/flaresolverr.yml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+            To use FlareSolverr, send POST requests to http://$$cap_appname.$$cap_root_domain/v1
+
+            Example curl request:
+            curl -L -X POST 'http://$$cap_appname.$$cap_root_domain/v1' \
+            -H 'Content-Type: application/json' \
+            --data-raw '{
+              "cmd": "request.get",
+              "url": "http://www.google.com/",
+              "maxTimeout": 60000
+            }'
```
Comment (githubsaturn) [inline public/v4/apps/flaresolverr.yml:103]: It's best to use Markdown here - use triple quote for the curl block ` ``` `

**Review by githubsaturn — COMMENTED** [review]
Thanks! Just a few minor comments.

**Comment by wajeht** [comment]
@githubsaturn fix request changes and it still works: https://fs.jaw.dev/

**Comment by githubsaturn** [comment]
Thanks!

**Pull request merged**